src/pages/details/DetailsPage.module.css

```diff
@@ -3,4 +3,9 @@
     flex-direction: row;
     justify-content: space-between;
     width: auto;
+}
+
+.newCarLoader {
```

Comment:
naming breaks with css naming conventions of using hypen as separator. Please switch from camel-case to hyphen-separation